app/services/curation_concerns/actors/actor_factory.rb

```diff
@@ -15,8 +15,8 @@ def self.stack_actors(curation_concern)
          ApplyOrderActor,
          InterpretVisibilityActor,
          GrantEditToDepositorActor,
-         InitializeWorkflowActor,
-         model_actor(curation_concern)]
+         model_actor(curation_concern),
+         InitializeWorkflowActor]
```

Comment:
I see you changed the order of these actors as well. Im curious as to what was going on here with the order of actions and what that might have been causing.